pages/videos.js

```diff
@@ -1,15 +1,83 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import HomeHero from '../components/HomeHero'
-import NavBar from '../components/NavBar'
+import Head from "next/head";
+import Image from "next/image";
+import HomeHero from "../components/HomeHero";
+import NavBar from "../components/NavBar";
 
 export default function Videos() {
-    return (
-      <>
-        <NavBar/>
-        <div className="flex justify-center content-center">
-        <iframe width="560" height="315" src="https://www.youtube.com/embed/QYvFllsI328"/>
-        </div>
-      </>
-    )
-  }
\ No newline at end of file
+  return (
+    <>
+      <NavBar />
+      <div className="flex justify-center content-center">
+        <iframe
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/QYvFllsI328"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        />
+      </div>
+      <br></br>
```

Comment:
Should be using width or other positioning options instead of br to reduce dom size